.release/ci.hcl

```diff
@@ -108,17 +108,12 @@ event "promote-production" {
     workflow = "promote-production"
   }
 
-  notification {
-    on = "always"
-  }
-}
-
-event "crt-hook-tfc-upload" {
-  depends = ["promote-production"]
-  action "crt-hook-tfc-upload" {
-    organization = "hashicorp"
-    repository = "terraform-releases"
-    workflow = "crt-hook-tfc-upload"
+  promotion-events  {
```

Comment:
Hrmm. I think you might have to rev your schema to v2. Our docs or my memory might not be clear on this. I vaguely recall this was a thing.

I'm going to loop in @emilymianeil to advise as she was on point for this feature and is going to be able to best vet the change.